### ISSUE TITLE
Reproductive interference

### ISSUE THREAD

**Issue by qgroom** [oborel/obo-relations#494]
Is reproductive interference covered in the Relation Ontology?
[https://en.wikipedia.org/wiki/Reproductive_interference](https://en.wikipedia.org/wiki/Reproductive_interference)

It is a biotic-biotic interaction, but doesn't fit under 'trophically interacts with' or 'symbiotically interacts with'

**Comment by nlharris** [comment]
Can anyone answer this?

**Comment by qgroom** [comment]
This is still a valid question to which no answer has been given. 
It is a pity not to even have it discussed.
Can it be reopened?

**Comment by wdduncan** [comment]
Yes. it can be reopened. But, do you have any suggestion on how to move forward?

**Comment by qgroom** [comment]
Well I would suggest it is added, but it comes under another category than 'trophically interacts with' or 'symbiotically interacts with', or it is a category of its own with different types of reproductive interference coming under that.

**Comment by wdduncan** [comment]
Sorry. I wasn't clear. Do you have a definition to propose?

**Comment by qgroom** [comment]
How about this definition?

`Reproductive interference refers to the detrimental interaction between individuals from different species that occurs during the process of finding and acquiring mates, resulting in a decrease in biological fitness for one or more of the individuals involved. `

That breaks down into several types...

- Misdirected Courtship
- Hybridization
- Sexual Harassment
- Gametic Wastage
- Competition for Mates
- Signal Jamming
- Niche Overlap
- Territorial Displacement

**Comment by wdduncan** [comment]
This seems like a good start. It needs to be discussed at the next RO call.

**Comment by bpeters42** [comment]
My general comment would be that if there is a new relationship for any possible type of interactions between organisms, RO will quickly swell to the size of GO. We had talked before to address that by creating design patterns - here something like 'organism A interacts with organism B in process C', where every C has participants A and B? I am stating this very crude, but it does feel wrong (and inefficient) for RO to discuss such very specific relations one-by-one in this forum.

**Comment by jhpoelen** [comment]
Hey @cmungall @bpeters42 thanks for contributing to the discussion started by @qgroom .

I'd say that if Quentin needs the term to describe in biotic interactions, then Quentin needs this term to describe biotic interactions.

As far as the more generalized approach to defining RO patterns instead of specific interaction type - I can see the benefit of only dealing with patterns, however, with this abstraction, the actual interaction type still needs to be defined somewhere. 

Also, we discussed this issue in great length as part of the reservoir host interaction type #757 . As far as I am concerned, no longer allowing so-called "short-cut" interactions would make RO less usable for mere mortals like myself. And, wouldn't you say that:

"I ate an apple"

is easier to understand (at least for me) than

"Organism A and organism B participated in a process C in which organism A ingested a part of organism B.  Organism A is classified as a prairie dwelling Homo sapiens, and organism B is classified as an apple."

I am hoping we can continue to strike a balance between elegant modeling and pragmatic use.

**Comment by jhpoelen** [comment]
As far as volunteering to create a pull request - I'd be happy to coach @qgroom through the process, and offer some reserved identifiers in my allocated term id range. 

You can find the edits I made to introduce reservoir host at https://github.com/oborel/obo-relations/commit/3102df2015078f11a87f073cbcb018c58b32daa1 . You can go the protege route (which I tried, but bailed out of), or you can just use vi (or other text editor) to add the new term in the accepted notation.

**Comment by qgroom** [comment]
Hi Jorrit, thanks for the offer! At the moment I realty wouldn't know where to begin. I will look through your `reservoir host` example to see if I can just follow that. Perhaps I'll start by documenting here what I think the solution is.

**Comment by qgroom** [comment]
So as I see it...

reproductively_interacts_with would be a subproperty of RO_0002574 (participates in a biotic-biotic interaction with)

Then the following would be subproperties of reproductively_interacts_with

- competition_for_mates_with
- gametic_wastage_with
- hybridization_with
- misdirected_courtship_with
- sexual_harassment_of
- signal_jamming_of

I left out niche overlap and territorial displacement that I mentioned earlier as these are not direct interactions.

**Comment by jhpoelen** [comment]
@qgroom Nice! Are you aiming to introduce all at once or start with the reproductively_interacts_with first?

**Comment by qgroom** [comment]
I guess one at a time would be safest, and easiest for a beginner like me.
I'll try to look at the documentation over the weekend, perhaps I can figure it out.

**Comment by qgroom** [comment]
Thanks!!! I'll follow up soon
